.github/workflows/dapps.yml

```diff
@@ -88,21 +135,21 @@ jobs:
         with:
           network: ${{ env.NETWORK }}
           devnet_solana_url: ${{ secrets.SOLANA_URL }}
-          proxy_ip: ${{ steps.prepare_terraform.outputs.proxy_ip }}
-          solana_ip: ${{ steps.prepare_terraform.outputs.solana_ip }}
-          proxy_url: ${{ github.event.inputs.proxy_url }}
-          solana_url: ${{ github.event.inputs.solana_url }}
-          faucet_url: ${{ github.event.inputs.faucet_url }}
-          network_id: ${{ github.event.inputs.network_id }}
+          proxy_ip: ${{ steps.prepare_terraform.outputs.proxy_ip || inputs.proxy_ip }}
+          solana_ip: ${{ steps.prepare_terraform.outputs.solana_ip || inputs.solana_ip }}
+          proxy_url: ${{ inputs.proxy_url }}
+          solana_url: ${{ inputs.solana_url }}
+          faucet_url: ${{ inputs.faucet_url }}
+          network_id: ${{ inputs.network_id }}
 
     outputs:
       network: ${{ env.NETWORK }}
       solana_url: ${{ steps.vars.outputs.solana_url }}
       proxy_url: ${{ steps.vars.outputs.proxy_url }}
       faucet_url:  ${{ steps.vars.outputs.faucet_url }}
       network_id: ${{ steps.vars.outputs.network_id }}
-      proxy_ip: ${{ steps.prepare_terraform.outputs.proxy_ip }}
-      solana_ip: ${{ steps.prepare_terraform.outputs.solana_ip }}
+      proxy_ip: ${{ steps.prepare_terraform.outputs.proxy_ip || inputs.proxy_ip }}
+      solana_ip: ${{ steps.prepare_terraform.outputs.solana_ip || inputs.solana_ip }}
```

Comment:
I think It is unnecessary, steps.prepare_terraform.outputs always returns something

Reply:
It was necessary when it was called outside from proxy repo, but now, after splitting the flow, it's not needed. Removed.